mis_builder/models/mis_report_instance.py

```diff
@@ -433,7 +433,14 @@ def _onchange_company(self):
                 ('id', 'child_of', self.company_id.id),
             ])
         else:
-            self.company_ids = self.company_id
+            self.company_ids = False
+
+    def _get_query_companies(self):
+        """ Return companies to query """
```

Comment:
I think there should be a ```api.multi``` with a ```self.ensure_one```.

Reply:
Why? It's just a helper method.

Reply:
Yes, but it must be called on a single recordset or it will crash. It will be the same behavior using the decorator, but IMO it's more explicit and we don't have to read all the method to know if it's a "static" or an object method.

Reply:
Ok, done.